aspnetcore/signalr/authn-and-authz.md

```diff
@@ -103,6 +95,25 @@ var connection = new HubConnectionBuilder()
 
 Windows Authentication is only supported by the browser client when using Microsoft Internet Explorer or Microsoft Edge.
 
+### Using Claims to Customize Identity Handling
+
+An app that authenticates users can derive SignalR user IDs from user claims. To specify how SignalR creates user IDs, implement `IUserIdProvider` and register the implementation.
+
+The sample code demonstrates how you would use claims to select the user's email address as the identifying property. 
```

Comment:
Sentence 1 "The sample code"
Sentence 2 "In the sample code"
Sentence 3 "The sample code"

The only suggestion I have is to only keep "sample code" for the first sentence and the other sentences should be fine without it?

Reply:
egads! agreed